hub/powertoys/video-conference-mute.md

```diff
@@ -45,7 +44,7 @@ The Video Conference Mute page in Settings provides the following options:
 | Camera overlay image | Select an image to that will be used as a placeholder when your camera is turned off. By default, a black screen will appear when your camera is turned off with this utility. |
 | Toolbar | Set the position where the _Microphone On, Camera On_ toolbar displays when toggled (default: top right corner). |
 | Show toolbar on | Select whether you prefer the toolbar to be displayed on the main monitor only (default) or on all monitors. |
-| Hide toolbar when both camera and microphone are unmuted | A checkbox is available to toggle this option. |
+| Hide toolbar when both camera and microphone are unmuted | |
```

Comment:
Is there some description that can be provided for this entry?

Reply:
I think the setting is pretty self-explanatory.